packages/at_persistence_secondary_server/lib/src/log/commitlog/sync/sync_keys_fetch_strategy.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:at_commons/at_commons.dart';
+import 'package:at_persistence_secondary_server/at_persistence_secondary_server.dart';
+import 'package:at_utils/at_utils.dart';
+
+abstract class SyncKeysFetchStrategy {
```

Comment:
Should we call it NamespaceHonoringKeyFetchStrategy?

Reply:
Honoring namespace is one of the checks done. regex match and skipping delete commits are the other two checks. In the future if we have any other checks not related to namespace we may have to rename this class